tests/integration_tests/test_config/resources.json

```diff
@@ -1,16 +1,15 @@
 
 {
     "comet": {
-      "cpus_on_node": 24,
-      "gpus_on_node": 0,
+      "cpus_per_node": 24,
```

Comment:
We use `cores_per_node` otherwise in RP - why `cpus_per_node` here?  I actually like that better, but it should be consistent maybe?

Reply:
Sorry. I will fix it.